docs/ref/csrf.txt

```diff
@@ -222,24 +222,34 @@ The CSRF protection is based on the following things:
 1. A CSRF cookie that is set to a random value (a session independent nonce, as
```

Comment:
"nonce" is technically not correct, I think. At least for the previous CSRF implementation as it was used more than once.